OpenRA.Mods.Common/Traits/Multipliers/VariedCostMultiplier.cs

```diff
@@ -0,0 +1,48 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Modifies the production cost of this actor for a specific queue or when a prerequisite is granted. " +
+		"Requires `VariedCostManager` on the world actor.")]
+	public class VariedCostMultiplierInfo : TraitInfo<VariedCostMultiplier>, IProductionCostModifierInfo, IRulesetLoaded
+	{
+		[Desc("Only apply this cost change if owner has these prerequisites.")]
+		public readonly string[] Prerequisites = { };
+
+		[Desc("Queues that this cost will apply.")]
+		public readonly HashSet<string> Queue = new HashSet<string>();
```

Comment:
`Queues` please, since this allows for multiple.

Reply:
Makes sense, but this is now inconsistent to
```salt
Buildable:
	Queue: Starport
```